src/os/ogc/wfs/wfsformatter.js

```diff
@@ -88,7 +87,8 @@ os.ogc.wfs.WFSFormatter.prototype.format = function(uri) {
   data += '>';
 
   var srs = /** @type {string} */ (wfsParams['srsname']) || os.ogc.defaultProjection;
-  data += '<wfs:Query srsName="' + srs + '" typeName="' + /** @type {string} */ (wfsParams['typename']) + '">';
+  data += '<wfs:Query srsName="' + srs + '" typeName="' +
```

Comment:
All changes in this file seem to only impact whitespace. Can they be dropped from the PR?

Reply:
Done